sdks/java/extensions/sql/jdbc/build.gradle

```diff
@@ -71,7 +68,35 @@ task shadowJarTest(type: Test, dependsOn: ":beam-sdks-java-extensions-sql-jdbc:s
   useJUnit { }
 }
 
+task endToEndTest(type: Test) {
+  group = "Verification"
+  def gcpProject = project.findProperty('gcpProject') ?: 'apache-beam-testing'
+  def gcsTempRoot = project.findProperty('gcsTempRoot') ?: 'gs://temp-storage-for-end-to-end-tests/'
+
+  // Disable Gradle cache (it should not be used because the IT's won't run).
+  outputs.upToDateWhen { false }
+
+  def pipelineOptions = [
+          "--project=${gcpProject}",
+          "--tempLocation=${gcsTempRoot}",
+          "--blockOnRun=false"]
+
+  systemProperty "beamTestPipelineOptions", JsonOutput.toJson(pipelineOptions)
```

Comment:
Your test isn't invoking a test pipeline, so this won't work.

Reply:
It actually works for TestPubsub where TestPipeline/TestPipelineOptions is invoked. So this configuration should be kept.